viewer/src/components/lineage/graphUtils.js

```diff
@@ -0,0 +1,181 @@
+import dagre from 'dagre';
+
+// Build nodes and edges from namedChildren
+export const buildGraph = (namedChildren) => {
```

Comment:
Surprised at the size of this file, no existing utils to make this easier to grok?